backend/tests/activityList.test.js

```diff
@@ -309,44 +317,6 @@ describe('Activity list controller', async () => {
       undefined
     ]);
   });
```

Comment:
Couple of tests being deleted .. are we losing coverage?

Reply:
No, I turned these into integration tests as I didn't duplicate test logic.